main/lsp/LSPTypecheckerCoordinator.cc

```diff
@@ -59,58 +64,22 @@ class TypecheckerTask final : public core::lsp::Task {
     }
 };
 
-// Special internal tasks that directly operate on `LSPTypechecker`. These are the only tasks that are allowed to
-// directly access `LSPTypechecker` (because they do special things). Thus, only `LSPTypecheckerCoordinator` is
-// allowed/able to create them.
-// TODO(jvilk): These implement `LSPTask` for the convenient `blockUntilComplete` method. Should we move that method
-// to `Task` directly?
-
-class InitializeTask : public LSPTask {
+class DangerousTypecheckerTask : public core::lsp::Task {
+    unique_ptr<LSPDangerousTypecheckerTask> task;
     LSPTypechecker &typechecker;
-    LSPFileUpdates updates;
-
-public:
-    InitializeTask(const LSPConfiguration &config, LSPTypechecker &typechecker, LSPFileUpdates updates)
-        : LSPTask(config, true), typechecker(typechecker), updates(move(updates)){};
-
-    void run(LSPTypecheckerDelegate &tcd) override {
-        typechecker.initialize(move(updates), tcd.workers);
-    }
-};
-
-class SlowPathTypecheckTask : public core::lsp::Task {
```

Comment:
SlowPathTypecheckerTask is now rolled into `notifications/SorbetWorkspaceEditTask.h`.